doc/classes/SceneState.xml

```diff
@@ -167,13 +167,6 @@
 				Returns the resource path to the represented [PackedScene].
 			</description>
 		</method>
-		<method name="is_node_instance_placeholder" qualifiers="const">
-			<return type="bool" />
-			<param index="0" name="idx" type="int" />
-			<description>
-				Returns [code]true[/code] if the node at [param idx] is an [InstancePlaceholder].
-			</description>
-		</method>
```

Comment:
Pipeline is failing because of this change. Since the method is still bound, its docs should stay.